scripts/javadoc/release.sh

```diff
@@ -0,0 +1,60 @@
+# Make sure the release_version was passed as an argument
+if [ $# -eq 0 ]; then
+    echo "Missing release_version argument."
+    echo "Usage: scripts/javadoc/release.sh major.minor.patch"
+    exit 1
+fi
+
+release_version=$1
+
+# Make sure the git repo is not dirty, since this script switches branches and commits changes.
+if [ "$(git status --porcelain)" ]; then
+    echo "Your git repo is dirty.  Commit or revert all changes before running this script."
+    exit 1
+fi
+
+# Make sure this script is run from the project root.
+if [[ ! -d "scripts" ]]
+then
+    echo "No scripts folder found.  This script must be run from the project root."
+    exit 1
+fi  
+
+# generate documents
+projectRoot=$(pwd)
+configFile="scripts/javadoc/ReleaseConfiguration.json"
+destinationPath="build/javadoc"
+sdkPath="$HOME/Library/Android/sdk/platforms/android-29/"
+
+## Make sure the Android SDK is installed at the expected place
+if [[ ! -f "${sdkPath}android.jar" ]]
+then
+    echo "Failed to find Android library at ${sdkPath}android.jar."
+    exit 1
+fi  
+
+python3 scripts/javadoc/generate.py "$configFile" "$projectRoot" "$destinationPath" "$sdkPath" "$release_version"
+
+# check out gh-pages and preserve old document
+git fetch
+git checkout gh-pages
+git checkout main scripts/javadoc/preserve_olddocument.sh
+release_version=$release_version bash scripts/javadoc/preserve_olddocument.sh
+
+# copy new document
+rm -rf docs/reference
+mkdir -p docs/reference
+cp -R build/javadoc/* docs/reference/
+
+# check in documents
+git add docs/reference
+git rm --cached  preserve_olddocument.sh
+git commit -m "AWS SDK for Android $release_version"
```

Comment:
"AWS SDK for Android" -> "Amplify Android"

Reply:
```suggestion
git commit -m "Amplify Android $release_version"
```